ui/src/RecentFilesDrawer/RecentFilesDrawer.tsx

```diff
@@ -252,6 +277,33 @@ export default function RecentFilesDrawer({ onclose }: Props) {
                   aria-label="new folder button"
                 />
               </HStack>
+
+              <Box style={{ position: 'relative' }}>
+                <IconSearch style={IconSearchStyle} />
+                <IconButton
+                  position="absolute"
+                  right="5px"
+                  width="20px"
+                  height="20px"
+                  top="50%"
+                  transform="translateY(-50%)"
+                  cursor="pointer"
+                  background="none"
+                  zIndex="100"
+                  icon={<IconX />}
+                  onClick={handleClearSearchValue}
+                  aria-label="clear input button"
+                />
```

Comment:
this should only appear when the input value is not empty, not appear all the time, also, make the size smaller, it's too big now